src/atoms/Flex/__tests__/Flex.native.test.js

```diff
@@ -1,15 +1,140 @@
 import React from 'react';
 import { render } from '@testing-library/react-native';
-import { View } from 'react-native';
+
+import View from '../../View';
 import Flex from '../index';
+import Space from '../../Space';
+import Size from '../../Size';
+
+const defaultFlexStyles = { flexDirection: 'column' };
+const defaultSpaceStyles = {};
+const defaultSizeStyles = {};
 
 describe('Renders <Flex /> correctly', () => {
-  it('snapshot testing', () => {
-    const { container } = render(
+  it('renders child of <Flex />(no styles) correctly', () => {
+    const { container, getByTestId } = render(
       <Flex>
-        <View />
+        <View testID={'flex-view'} />
+      </Flex>,
+    );
+    const childView = getByTestId('flex-view');
+    expect(childView.props.style).toMatchObject([defaultFlexStyles]);
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('<Flex /> and <Space /> together plays properly', () => {
```

Comment:
```suggestion
describe('<Flex /> and <Space />', () => {
```

Reply:
Trying to be as descriptive and that also using layman language.